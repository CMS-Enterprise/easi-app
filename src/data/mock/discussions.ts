import {
  SystemIntakeGRBReviewDiscussionFragment,
  SystemIntakeGRBReviewerRole,
  SystemIntakeGRBReviewerVotingRole
} from 'gql/gen/graphql';

const mockDiscussions = (
  systemIntakeID: string = '0d191c27-4de3-46d8-9ded-c2e9c6b3fbd6'
): SystemIntakeGRBReviewDiscussionFragment[] => [
  {
    __typename: 'SystemIntakeGRBReviewDiscussion',
    initialPost: {
      __typename: 'SystemIntakeGRBReviewDiscussionPost',
      id: '882357e4-c0b0-44ef-b749-f71879ad7878',
      content:
        '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>',
      votingRole: SystemIntakeGRBReviewerVotingRole.VOTING,
      grbRole: SystemIntakeGRBReviewerRole.SUBJECT_MATTER_EXPERT,
      createdByUserAccount: {
        __typename: 'UserAccount',
        id: '034fa2b3-00ff-4ec6-857e-75291a59df74',
        commonName: 'Adeline Aarons'
      },
      systemIntakeID,
      createdAt: '2024-11-12T21:59:07.368862Z'
    },
    replies: [
      {
        __typename: 'SystemIntakeGRBReviewDiscussionPost',
        id: '4099dbb7-2752-4bf9-a3e1-da225ceb9fae',
        content:
          '<p>Nisi nobis consectetur voluptatem neque tempore. Ea nihil sed beatae?</p>',
        votingRole: SystemIntakeGRBReviewerVotingRole.NON_VOTING,
        grbRole: SystemIntakeGRBReviewerRole.CMCS_REP,
        createdByUserAccount: {
          __typename: 'UserAccount',
          id: '909a7888-4d6f-4bbf-9b9d-71ec1e2c3068',
          commonName: 'Jerry Seinfeld'
        },
        systemIntakeID,
        createdAt: '2024-11-13T21:59:07.368862Z'
      },
      {
        __typename: 'SystemIntakeGRBReviewDiscussionPost',
        id: '47b0081d-de33-4514-b68f-a7e2bbf2610f',
        content:
          '<p>Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?</p>',
        votingRole: SystemIntakeGRBReviewerVotingRole.VOTING,
        grbRole: SystemIntakeGRBReviewerRole.CO_CHAIR_CIO,
        createdByUserAccount: {
          __typename: 'UserAccount',
          id: '601d52be-7baa-4b45-91cd-88b4a5935c3f',
          commonName: 'Cosmo Kramer'
        },
        systemIntakeID,
        createdAt: '2024-11-13T21:59:07.368862Z'
      }
    ]
  }
];

export default mockDiscussions;
