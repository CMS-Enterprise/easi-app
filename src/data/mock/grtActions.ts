import GetCedarContactsQuery from 'queries/GetCedarContactsQuery';
import { GetSystemIntakeContactsQuery } from 'queries/SystemIntakeContactsQueries';

const systemIntakeId = 'a4158ad8-1236-4a55-9ad5-7e15a5d49de2';

const requester = {
  systemIntakeId,
  role: 'Requester',
  euaUserId: 'SF13',
  commonName: 'Jerry Seinfeld',
  component: 'Center for Medicaid and CHIP Services',
  email: 'sf13@local.fake',
  id: 'dadffca7-35ef-4498-974b-e5fe95ff028e'
};

const businessOwner = {
  systemIntakeId,
  role: 'Business Owner',
  euaUserId: 'SF13',
  commonName: 'Jerry Seinfeld',
  component: 'Center for Medicaid and CHIP Services',
  email: 'sf13@local.fake',
  id: 'b25bd13b-72a0-4d0e-b5be-7852a1a8259d'
};

const productManager = {
  systemIntakeId,
  euaUserId: 'KR14',
  role: 'Product Manager',
  commonName: 'Cosmo Kramer',
  component: 'Center for Program Integrity',
  email: 'kr14@local.fake',
  id: '' // Leave out ID so contact shows as unverified
};

const isso = {
  systemIntakeId,
  euaUserId: 'WXYZ',
  role: 'ISSO',
  commonName: 'John Smith',
  component: 'CMS Wide',
  email: 'wxyz@local.fake',
  id: '346eefa9-c635-4c0b-bc29-26f272c33d0c'
};

const systemIntake = {
  id: systemIntakeId,
  euaUserId: requester.euaUserId,
  adminLead: '',
  businessNeed: 'Test business need',
  businessSolution: 'Test business solution',
  businessOwner: {
    component: businessOwner.component,
    name: businessOwner.commonName
  },
  contract: {
    contractor: 'Contractor Name',
    endDate: {
      day: '31',
      month: '12',
      year: '2023'
    },
    hasContract: 'HAVE_CONTRACT',
    startDate: {
      day: '1',
      month: '1',
      year: '2021'
    },
    vehicle: 'Sole source',
    number: '123456-7890'
  },
  costs: {
    isExpectingIncrease: 'YES',
    expectedIncreaseAmount: '10 million dollars'
  },
  annualSpending: {
    currentAnnualSpending: 'Test Current Annual Spending',
    plannedYearOneSpending: 'Test Planned Year One Spending'
  },
  currentStage: 'I have done some initial research',
  decisionNextSteps: '',
  grbDate: null,
  grtDate: null,
  grtFeedbacks: [],
  governanceTeams: {
    isPresent: true,
    teams: []
  },
  isso: {
    isPresent: true,
    component: isso.component,
    name: isso.commonName
  },
  existingFunding: true,
  fundingSources: [{ fundingNumber: '123456', source: 'Research' }],
  lcid: '',
  lcidExpiresAt: null,
  lcidScope: '',
  lcidCostBaseline: null,
  needsEaSupport: true,
  productManager: {
    component: productManager.component,
    name: productManager.commonName
  },
  rejectionReason: '',
  requester: {
    component: requester.component,
    email: requester.email,
    name: requester.commonName
  },
  requestName: 'TACO',
  requestType: 'NEW',
  status: 'INTAKE_SUBMITTED',
  createdAt: null,
  submittedAt: null,
  updatedAt: null,
  archivedAt: null,
  decidedAt: null,
  businessCaseId: null,
  lastAdminNote: null,
  grtReviewEmailBody: '',
  hasUiChanges: true,
  documents: []
};

export const mockData = {
  contacts: {
    requester,
    businessOwner,
    productManager,
    isso
  },
  systemIntake
};
export const contactQueries = {
  getSystemIntakeContactsQuery: {
    request: {
      query: GetSystemIntakeContactsQuery,
      variables: {
        id: systemIntakeId
      }
    },
    result: {
      data: {
        systemIntakeContacts: {
          systemIntakeContacts: [requester, businessOwner, isso]
        }
      }
    }
  },
  getCedarContactsQuery: {
    request: {
      query: GetCedarContactsQuery,
      variables: {
        commonName: productManager.commonName
      }
    },
    result: {
      data: {
        cedarPersonsByCommonName: [
          {
            commonName: productManager.commonName,
            email: productManager.email,
            euaUserId: productManager.euaUserId
          }
        ]
      }
    }
  }
};

export const grtActions = {
  'not-it-request': {
    heading: 'Not an IT governance request',
    view: 'grt-submit-action-view'
  },
  'need-biz-case': {
    heading: 'Request a draft business case',
    view: 'grt-submit-action-view'
  },
  'provide-feedback-need-biz-case': {
    heading: 'Provide GRT Feedback and progress to business case',
    view: 'provide-feedback-biz-case'
  },
  'provide-feedback-keep-draft': {
    heading: 'Provide GRT feedback and keep working on draft business case',
    view: 'provide-feedback-biz-case'
  },
  'provide-feedback-need-final': {
    heading: 'Provide GRT feedback and request final business case for GRB',
    view: 'provide-feedback-biz-case'
  },
  'ready-for-grt': {
    heading: 'Mark as ready for GRT',
    view: 'grt-submit-action-view'
  },
  'ready-for-grb': {
    heading: 'Mark as ready for GRB',
    view: 'ready-for-grb'
  },
  'biz-case-needs-changes': {
    heading: 'Business case needs changes and is not ready for GRT',
    view: 'grt-submit-action-view'
  },
  'no-governance': {
    heading: 'Close project',
    view: 'grt-submit-action-view'
  },
  'send-email': {
    heading: 'Send email',
    view: 'grt-submit-action-view'
  },
  'guide-received-close': {
    heading: 'Decomission guide received. Close the request',
    view: 'grt-submit-action-view'
  },
  'not-responding-close': {
    heading: 'Project team not responding. Close the request',
    view: 'grt-submit-action-view'
  },
  'issue-lcid': {
    heading: 'Approve request and issue Lifecycle ID',
    view: 'issue-lcid'
  },
  'extend-lcid': {
    heading: 'Extend Lifecycle ID',
    view: 'extend-lcid'
  },
  'not-approved': {
    heading: 'Business case not approved',
    view: 'not-approved'
  }
};
