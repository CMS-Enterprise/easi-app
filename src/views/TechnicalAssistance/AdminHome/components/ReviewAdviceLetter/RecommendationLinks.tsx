import React from 'react';
import { Link } from '@trussworks/react-uswds';
import classNames from 'classnames';

import formatUrl from 'utils/formatUrl';

type RecommendationLinksProps = {
  links: string[];
  className?: string;
};

export default function RecommendationLinks({
  links,
  className
}: RecommendationLinksProps) {
  return (
    <ul className={classNames('usa-list', 'usa-list--unstyled', className)}>
      {links.map((link, index) => {
        const linkString = formatUrl(link);

        const url = new URL(`https://${linkString}`);

        const isExternal: boolean = url.host !== 'easi.cms.gov';

        return (
          <li
            key={`link-${index}`} // eslint-disable-line react/no-array-index-key
          >
            <Link
              aria-label={`Open ${link} in a new tab`}
              target="_blank"
              rel="noopener noreferrer"
              className="display-block margin-top-05"
              // If EASi url, link to relative path
              href={isExternal ? url.href : url.pathname}
              // Display external link icon if not EASi url
              {...(isExternal && { variant: 'external' })}
            >
              {linkString}
            </Link>
          </li>
        );
      })}
    </ul>
  );
}
