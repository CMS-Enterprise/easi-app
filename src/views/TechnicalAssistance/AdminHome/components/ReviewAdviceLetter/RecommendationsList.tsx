import React, { useState } from 'react';
import { useTranslation } from 'react-i18next';
import { Button, ButtonGroup } from '@trussworks/react-uswds';
import classNames from 'classnames';

import { GetTrbAdviceLetter_trbRequest_adviceLetter_recommendations as TRBRecommendation } from 'queries/types/GetTrbAdviceLetter';

import RemoveRecommendationModal from '../RemoveRecommendationModal/Index';

import RecommendationLinks from './RecommendationLinks';

export type EditRecommendationProp = {
  onClick: (recommendation: TRBRecommendation) => void;
  text?: string;
};

export type RemoveRecommendationProp = {
  onClick: (recommendation: TRBRecommendation) => void;
  text?: string;
};

type RecommendationsListProps = {
  type: 'form' | 'admin';
  recommendations: TRBRecommendation[];
  edit?: EditRecommendationProp;
  remove?: RemoveRecommendationProp;
  className?: string;
};

export default function RecommendationsList({
  type,
  recommendations,
  edit,
  remove,
  className
}: RecommendationsListProps) {
  const { t } = useTranslation('technicalAssistance');

  const [
    recommendationToRemove,
    setRecommendationToRemove
  ] = useState<TRBRecommendation | null>(null);

  return (
    <>
      {remove && (
        <RemoveRecommendationModal
          modalProps={{
            isOpen: !!recommendationToRemove,
            closeModal: () => setRecommendationToRemove(null)
          }}
          handleDelete={() =>
            recommendationToRemove && remove.onClick(recommendationToRemove)
          }
        >
          <p>
            {t('adviceLetterForm.modal.removingTitle', {
              title: recommendationToRemove?.title
            })}
          </p>
        </RemoveRecommendationModal>
      )}
      <ul
        className={classNames(
          'usa-list usa-list--unstyled',
          { 'grid-row grid-gap-lg': type === 'form' },
          className
        )}
      >
        {recommendations.map(recommendation => {
          const {
            title,
            id,
            links,
            recommendation: description
          } = recommendation;

          return (
            <li
              key={id}
              className={classNames({
                'desktop:grid-col-6': type === 'form',
                'bg-base-lightest padding-x-4 padding-y-1 padding-bottom-4 margin-bottom-3':
                  type === 'admin'
              })}
            >
              {type === 'admin' ? (
                <h3 className="margin-bottom-1">{title}</h3>
              ) : (
                <h4 className="margin-bottom-1">{title}</h4>
              )}

              <p className="margin-y-1">{description}</p>

              {links.length > 0 && (
                <>
                  {type === 'admin' && (
                    <p className="text-bold margin-bottom-0 margin-top-3">
                      {t('adviceLetter.resources')}
                    </p>
                  )}
                  <RecommendationLinks links={links} />
                </>
              )}

              {
                // Show action buttons if edit or remove onClick props are provided
                (edit || remove) && (
                  <ButtonGroup className="margin-top-2">
                    {edit && (
                      <Button
                        type="button"
                        onClick={() => edit.onClick(recommendation)}
                        unstyled
                      >
                        {t(edit.text || 'adviceLetterForm.editRecommendation')}
                      </Button>
                    )}
                    {remove && (
                      <Button
                        type="button"
                        className="text-secondary margin-left-1"
                        onClick={() =>
                          setRecommendationToRemove(recommendation)
                        }
                        unstyled
                      >
                        {t(
                          remove.text || 'adviceLetterForm.removeRecommendation'
                        )}
                      </Button>
                    )}
                  </ButtonGroup>
                )
              }
            </li>
          );
        })}
      </ul>
    </>
  );
}
