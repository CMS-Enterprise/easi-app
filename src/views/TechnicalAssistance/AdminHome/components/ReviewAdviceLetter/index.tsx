import React from 'react';
import { useTranslation } from 'react-i18next';
import { Grid } from '@trussworks/react-uswds';
import classNames from 'classnames';
import { DateTime } from 'luxon';

import UswdsReactLink from 'components/LinkWrapper';
import { RichTextViewer } from 'components/RichTextEditor';
import SectionWrapper from 'components/shared/SectionWrapper';
import {
  GetTrbAdviceLetter_trbRequest as TrbRequest,
  GetTrbAdviceLetter_trbRequest_adviceLetter as AdviceLetter
} from 'queries/types/GetTrbAdviceLetter';
import { TRBRecommendation } from 'queries/types/TRBRecommendation';
import { formatDateLocal } from 'utils/date';

import RecommendationsList from '../RecommendationsList';

type ReviewAdviceLetterProps = {
  adviceLetter: AdviceLetter;
  trbRequestId: string;
  trbRequest?: TrbRequest;
  requesterString?: string;
  showSectionEditLinks?: boolean;
  recommendationActions?: {
    edit?: (recommendation: TRBRecommendation) => void;
    remove?: (recommendation: TRBRecommendation) => void;
    setReorderError?: (error: string | null) => void;
  };
  showDateSent?: boolean;
  showSectionBorders?: boolean;
  editable?: boolean;
  publicView?: boolean;
  className?: string;
};

/**
 * Displays advice letter for review
 */
const ReviewAdviceLetter = ({
  adviceLetter,
  trbRequestId,
  trbRequest,
  requesterString,
  showSectionEditLinks = false,
  recommendationActions,
  showDateSent = true,
  showSectionBorders = true,
  editable = true,
  publicView = false,
  className
}: ReviewAdviceLetterProps) => {
  const { t } = useTranslation('technicalAssistance');

  const { recommendations } = adviceLetter;

  return (
    <div className={className}>
      {/* Thank you text for PDF version */}
      <p className="easi-only-print">{t('adviceLetter.thankYou')}</p>

      {!publicView && trbRequest && (
        <div className="easi-only-print">
          {/* Project title */}
          <p className="text-bold margin-bottom-0">{t('basic.labels.name')}</p>
          <p className="margin-top-1">{trbRequest.name}</p>

          {/* Request Type */}
          <p className="text-bold margin-bottom-0">
            {t('adminHome.requestType')}
          </p>
          <p className="margin-top-1">
            {t(`requestType.type.${trbRequest.type}.heading`)}
          </p>

          {/* Requester */}
          <p className="text-bold margin-bottom-0">
            {t('adminHome.requester')}
          </p>
          <p className="margin-top-1">{requesterString}</p>

          {/* Submission date */}
          <p className="text-bold margin-bottom-0">
            {t('adminHome.submissionDate')}
          </p>
          <p className="margin-top-1">
            {formatDateLocal(trbRequest.createdAt, 'MMMM d, yyyy')}
          </p>
        </div>
      )}

      {!publicView && (
        <Grid row gap className="margin-bottom-neg-2">
          {/* Date sent */}
          {showDateSent && (
            <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
              <p className="text-bold margin-y-0 easi-no-print">
                {t('adviceLetter.sendDate')}
              </p>

              <p className="margin-top-1 easi-no-print">
                {adviceLetter.dateSent
                  ? formatDateLocal(adviceLetter.dateSent, 'MMMM d, yyyy')
                  : t('adviceLetter.notYetSent')}
              </p>
            </Grid>
          )}

          {/* Consult session date */}
          {trbRequest && trbRequest.consultMeetingTime && (
            <Grid tablet={{ col: 12 }} desktop={{ col: 6 }}>
              <p className="text-bold margin-y-0">
                {t('adviceLetter.consultSessionDate')}
              </p>
              <p className="margin-top-1">
                {t('adminHome.consultDate', {
                  date: formatDateLocal(
                    trbRequest.consultMeetingTime,
                    'MM/dd/yyyy'
                  ),
                  time: DateTime.fromISO(
                    trbRequest.consultMeetingTime
                  ).toLocaleString(DateTime.TIME_SIMPLE)
                })}
              </p>
            </Grid>
          )}
        </Grid>
      )}

      {/* What we heard / meeting summary */}
      <SectionWrapper
        borderTop={showSectionBorders}
        className={classNames({
          'margin-top-6 padding-top-1': showSectionBorders,
          'margin-top-5': !showSectionBorders
        })}
      >
        <h2 className="margin-bottom-1">{t('adviceLetter.whatWeHeard')}</h2>

        {showSectionEditLinks && (
          <UswdsReactLink
            to={`/trb/${trbRequestId}/advice/summary`}
            className="display-block margin-bottom-5"
          >
            {t('check.edit')}
          </UswdsReactLink>
        )}

        {!publicView && (
          <p className="easi-no-print text-bold margin-top-4 margin-bottom-0">
            {t('adviceLetter.meetingSummary')}
          </p>
        )}

        {adviceLetter.meetingSummary && (
          <RichTextViewer
            className="margin-top-1"
            value={adviceLetter.meetingSummary}
          />
        )}
      </SectionWrapper>

      {/* Recommendations */}
      <SectionWrapper
        borderTop={showSectionBorders}
        className={classNames({
          'margin-top-6 padding-top-1': showSectionBorders,
          'margin-top-5': !showSectionBorders
        })}
      >
        <h2 className="margin-bottom-1">{t('adviceLetter.whatWeRecommend')}</h2>

        {showSectionEditLinks && (
          <UswdsReactLink
            to={`/trb/${trbRequestId}/advice/recommendations`}
            className="display-block margin-bottom-2"
          >
            {t('check.edit')}
          </UswdsReactLink>
        )}

        {
          // If no recommendations, return text
          recommendations.length === 0 ? (
            <p className="margin-top-4">{t('adviceLetter.notSpecified')}</p>
          ) : (
            <RecommendationsList
              recommendations={recommendations}
              trbRequestId={trbRequestId}
              className="margin-top-4"
              editable={editable}
              {...recommendationActions}
            />
          )
        }
      </SectionWrapper>

      {/* Next steps */}
      <SectionWrapper
        borderTop={showSectionBorders}
        className={classNames({
          'margin-top-6 padding-top-1': showSectionBorders,
          'margin-top-5': !showSectionBorders
        })}
      >
        <h2 className="margin-bottom-1">{t('adviceLetter.nextSteps')}</h2>
        {showSectionEditLinks && (
          <UswdsReactLink
            to={`/trb/${trbRequestId}/advice/next-steps`}
            className="display-block margin-bottom-5"
          >
            {t('check.edit')}
          </UswdsReactLink>
        )}

        <p className="text-bold margin-top-4 margin-bottom-1">
          {t('adviceLetter.nextSteps')}
        </p>

        {adviceLetter.nextSteps ? (
          <RichTextViewer
            className="margin-top-1"
            value={adviceLetter.nextSteps}
          />
        ) : (
          <p className="margin-top-1 line-height-body-5">
            {t('adviceLetter.notSpecified')}
          </p>
        )}

        {/* Follow up */}
        <p className="text-bold margin-top-3 margin-bottom-1">
          {t('adviceLetter.followup')}
        </p>

        <p className="margin-top-1 line-height-body-5">
          {adviceLetter.isFollowupRecommended
            ? t(`Yes, ${adviceLetter.followupPoint}`)
            : t('adviceLetterForm.notNecessary')}
        </p>
      </SectionWrapper>
    </div>
  );
};

export default ReviewAdviceLetter;
