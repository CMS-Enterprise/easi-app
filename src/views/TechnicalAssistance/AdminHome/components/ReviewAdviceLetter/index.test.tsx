import React from 'react';
import { render } from '@testing-library/react';

import { adviceLetter } from 'data/mock/trbRequest';

import ReviewAdviceLetter from '.';

describe('TRB Advice Letter review component', () => {
  it('Matches the snapshot', async () => {
    const { getByText, asFragment } = render(
      <ReviewAdviceLetter
        adviceLetter={adviceLetter}
        trbRequestId="8d8bbac8-4ce5-43c5-9d47-a698be37d152"
      />
    );

    // Advice letter renders
    expect(getByText('Meeting summary text')).toBeInTheDocument();

    // Recommendation letter renders
    expect(getByText('Recommendation 1')).toBeInTheDocument();

    // Snapshot
    expect(asFragment()).toMatchSnapshot();
  });
});
