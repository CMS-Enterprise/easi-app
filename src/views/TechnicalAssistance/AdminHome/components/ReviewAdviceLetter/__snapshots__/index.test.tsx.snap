// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TRB Advice Letter review component > Matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    <p
      class="easi-only-print"
    >
      Thank you for attending the TRB consult session!
    </p>
    <div
      class="grid-row grid-gap margin-bottom-neg-2"
      data-testid="grid"
    >
      <div
        class="tablet:grid-col-12 desktop:grid-col-6"
        data-testid="grid"
      >
        <p
          class="text-bold margin-y-0 easi-no-print"
        >
          Send date
        </p>
        <p
          class="margin-top-1 easi-no-print"
        >
          Not yet sent
        </p>
      </div>
    </div>
    <div
      class="easi-section easi-section__border-top margin-top-6 padding-top-1"
      data-testid="section-wrapper"
    >
      <h2
        class="margin-bottom-1"
      >
        What we heard
      </h2>
      <p
        class="easi-no-print text-bold margin-top-4 margin-bottom-0"
      >
        Meeting summary
      </p>
      <div
        class="easi-toast easi-toast-viewer margin-top-1"
      >
        <div>
          <div
            class="toastui-editor-contents"
            style="overflow-wrap: break-word;"
          >
            <p
              data-nodeid="1"
            >
              Meeting summary text
            </p>
            

          </div>
        </div>
      </div>
    </div>
    <div
      class="easi-section easi-section__border-top margin-top-6 padding-top-1"
      data-testid="section-wrapper"
    >
      <h2
        class="margin-bottom-1"
      >
        What we recommend
      </h2>
      <div
        class="margin-top-4"
      >
        <div
          class="usa-alert usa-alert--info usa-alert--slim margin-bottom-4"
          data-testid="alert"
        >
          <div
            class="usa-alert__body"
          >
            <p
              class="usa-alert__text"
            >
              If you have two or more recommendations, you can use arrows on the left side of each recommendation to reorder them.
            </p>
          </div>
        </div>
        <ul
          class="usa-list usa-list--unstyled"
        >
          <li
            class="margin-bottom-3"
            data-testid="recommendations_list-item"
          >
            <div
              class="bg-base-lightest padding-top-2 padding-bottom-105 padding-left-105 padding-right-3 display-flex"
            >
              <div
                class="margin-right-2 display-flex flex-column flex-align-center line-height-body-1"
                data-testid="reorder-controls"
              >
                <button
                  aria-label="Increase recommendation sort order"
                  class="usa-button usa-button--unstyled height-3"
                  data-testid="button"
                  type="button"
                >
                  <svg
                    class="usa-icon usa-icon--size-3 text-primary"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="m7 14 5-5 5 5z"
                    />
                  </svg>
                </button>
                <span
                  data-testid="order-index"
                >
                  1
                </span>
                <button
                  aria-label="Decrease recommendation sort order"
                  class="usa-button usa-button--unstyled height-3"
                  data-testid="button"
                  type="button"
                >
                  <svg
                    class="usa-icon usa-icon--size-3 text-primary"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="m7 10 5 5 5-5z"
                    />
                  </svg>
                </button>
              </div>
              <div
                class="width-full padding-bottom-2 padding-top-105"
              >
                <h3
                  class="margin-top-0 margin-bottom-05"
                >
                  Recommendation 1
                </h3>
                <div
                  class="easi-toast easi-toast-viewer margin-top-1"
                >
                  <div>
                    <div
                      class="toastui-editor-contents"
                      style="overflow-wrap: break-word;"
                    >
                      <p
                        data-nodeid="3"
                      >
                        This is the recommendation text
                      </p>
                      

                    </div>
                  </div>
                </div>
                <p
                  class="text-bold margin-bottom-0 margin-top-2"
                >
                  Resources
                </p>
                <ul
                  class="usa-list usa-list--unstyled margin-bottom-05"
                >
                  <li>
                    <a
                      aria-label="Open easi.cms.gov in a new tab"
                      class="usa-link"
                      href="/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      easi.cms.gov
                    </a>
                  </li>
                  <li>
                    <a
                      aria-label="Open https://google.com in a new tab"
                      class="usa-link usa-link--external"
                      href="https://google.com/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      google.com
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </li>
          <li
            class="margin-bottom-3"
            data-testid="recommendations_list-item"
          >
            <div
              class="bg-base-lightest padding-top-2 padding-bottom-105 padding-left-105 padding-right-3 display-flex"
            >
              <div
                class="margin-right-2 display-flex flex-column flex-align-center line-height-body-1"
                data-testid="reorder-controls"
              >
                <button
                  aria-label="Increase recommendation sort order"
                  class="usa-button usa-button--unstyled height-3"
                  data-testid="button"
                  type="button"
                >
                  <svg
                    class="usa-icon usa-icon--size-3 text-primary"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="m7 14 5-5 5 5z"
                    />
                  </svg>
                </button>
                <span
                  data-testid="order-index"
                >
                  2
                </span>
                <button
                  aria-label="Decrease recommendation sort order"
                  class="usa-button usa-button--unstyled height-3"
                  data-testid="button"
                  type="button"
                >
                  <svg
                    class="usa-icon usa-icon--size-3 text-primary"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="m7 10 5 5 5-5z"
                    />
                  </svg>
                </button>
              </div>
              <div
                class="width-full padding-bottom-2 padding-top-105"
              >
                <h3
                  class="margin-top-0 margin-bottom-05"
                >
                  Recommendation 2
                </h3>
                <div
                  class="easi-toast easi-toast-viewer margin-top-1"
                >
                  <div>
                    <div
                      class="toastui-editor-contents"
                      style="overflow-wrap: break-word;"
                    >
                      <p
                        data-nodeid="5"
                      >
                        This is the recommendation text
                      </p>
                      

                    </div>
                  </div>
                </div>
                <p
                  class="text-bold margin-bottom-0 margin-top-2"
                >
                  Resources
                </p>
                <ul
                  class="usa-list usa-list--unstyled margin-bottom-05"
                >
                  <li>
                    <a
                      aria-label="Open easi.cms.gov in a new tab"
                      class="usa-link"
                      href="/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      easi.cms.gov
                    </a>
                  </li>
                  <li>
                    <a
                      aria-label="Open cms.gov in a new tab"
                      class="usa-link usa-link--external"
                      href="https://cms.gov/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      cms.gov
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </li>
          <li
            class="margin-bottom-3"
            data-testid="recommendations_list-item"
          >
            <div
              class="bg-base-lightest padding-top-2 padding-bottom-105 padding-left-105 padding-right-3 display-flex"
            >
              <div
                class="margin-right-2 display-flex flex-column flex-align-center line-height-body-1"
                data-testid="reorder-controls"
              >
                <button
                  aria-label="Increase recommendation sort order"
                  class="usa-button usa-button--unstyled height-3"
                  data-testid="button"
                  type="button"
                >
                  <svg
                    class="usa-icon usa-icon--size-3 text-primary"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="m7 14 5-5 5 5z"
                    />
                  </svg>
                </button>
                <span
                  data-testid="order-index"
                >
                  3
                </span>
                <button
                  aria-label="Decrease recommendation sort order"
                  class="usa-button usa-button--unstyled height-3"
                  data-testid="button"
                  type="button"
                >
                  <svg
                    class="usa-icon usa-icon--size-3 text-primary"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="m7 10 5 5 5-5z"
                    />
                  </svg>
                </button>
              </div>
              <div
                class="width-full padding-bottom-2 padding-top-105"
              >
                <h3
                  class="margin-top-0 margin-bottom-05"
                >
                  Recommendation 3
                </h3>
                <div
                  class="easi-toast easi-toast-viewer margin-top-1"
                >
                  <div>
                    <div
                      class="toastui-editor-contents"
                      style="overflow-wrap: break-word;"
                    >
                      <p
                        data-nodeid="7"
                      >
                        This is the recommendation text
                      </p>
                      

                    </div>
                  </div>
                </div>
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div
      class="easi-section easi-section__border-top margin-top-6 padding-top-1"
      data-testid="section-wrapper"
    >
      <h2
        class="margin-bottom-1"
      >
        Next steps
      </h2>
      <p
        class="text-bold margin-top-4 margin-bottom-1"
      >
        Next steps
      </p>
      <div
        class="easi-toast easi-toast-viewer margin-top-1"
      >
        <div>
          <div
            class="toastui-editor-contents"
            style="overflow-wrap: break-word;"
          >
            <p
              data-nodeid="9"
            >
              These are the next steps
            </p>
            

          </div>
        </div>
      </div>
      <p
        class="text-bold margin-top-3 margin-bottom-1"
      >
        Should the team return for a follow-up session?
      </p>
      <p
        class="margin-top-1 line-height-body-5"
      >
        Yes, Six months from now
      </p>
    </div>
  </div>
</DocumentFragment>
`;
